app.js

```diff
@@ -42,65 +50,114 @@ const createMultipartSdp = (sdp, {
   const sipSessionId = originalInvite.get('Call-ID');
   const {originator = 'unknown', carrier = 'unknown'} = originalInvite.locals;
 
-  const x = `--uniqueBoundary
-Content-Disposition: session;handling=required
+  var x ;
+
+   
+if (process.env.JAMBONES_SIPREC_TYPE=="SMART_TAP"){
```

Comment:
I think it would be nice to refactor the function of generating the body into a function in a lib/utils.js file in order to keep this file more readable and simple.  If we add more in the future this would be important